server/filestore.go

```diff
@@ -5317,7 +5317,7 @@ func (fs *fileStore) expireMsgs() {
 	defer fs.mu.Unlock()
 
 	// TODO: Not great that we're holding the lock here, but the timed hash wheel isn't thread-safe.
-	var nextTTL int64
+	nextTTL := int64(math.MaxInt64)
```

Comment:
was wondering if 

```var nextTTL int64 = math.MaxInt64``` 

is better, but I think your variant is shorter and more idiomatic.